src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -991,6 +999,123 @@ static boolean use43Wrapper() {
     static String escapeSingleQuotes(String name) {
         return name.replace("'", "''");
     }
+
+    static ResultSet executeSDPEv2(PreparedStatement stmt, String userSql, String preparedTypeDefinitions,
```

Comment:
I don't think Util is a good place for this. utils contain very generic types utils that are used everywhere, this is very specific for AE

Reply:
We already have this weird "shoudHonorAEFor..." method, it doesn't seem like we actually have a specific methodology for what goes into Utils. I was looking for something like AEUtils but that doesn't exist.